src/azul/indexer/__init__.py

```diff
@@ -456,6 +456,27 @@ def reject_joiner(self):
         self._reject_joiner(self.manifest)
         self._reject_joiner(self.metadata_files)
 
+    @classmethod
+    def extension(cls) -> str:
```

Comment:
I split this change across two fixups, so that the `.hca` component is added in the commit that introduces the divergence of the canned bundle formats for HCA and AnVIL.